### DISCUSSION TITLE
27.45XCH was stolen from me

### DISCUSSION THREAD

**Discussion by Laushd** [Chia-Network/chia-blockchain#14823] [Support]
Hello. 27.45 XCH was stolen from me on March 7th. The official Chia wallet. It is almost always turned off. The last phrase from him is only in my paper notebook. Stolen in two transactions, 27XCH and 0.45 XCH. Can you help me get them back?

Address of the thief:  xch1mgda9ll4kst0fh30847tdurjwq9rj6vxumqhxu3xf6r2xt864hdq8s99ut
Confirmed at Height | 3346415
and 3346429
https://alltheblocks.net/chia/address/xch1mgda9ll4kst0fh30847tdurjwq9rj6vxumqhxu3xf6r2xt864hdq8s99ut

I have been saving these coins for many years, and they were stolen.

Please help punish the thief and return the coins. I can prove the origin of every mojo.

Thanks! I am waiting for your answer! 
Dmitry.

**Comment by shaneo257** [comment]
Unfortunately, we won't be able to help you get the coins back

**Reply by Laushd** [reply]
> Unfortunately, we won't be able to help you get the coins back

I'm not the first one to be attacked by hackers. https://github.com/Chia-Network/chia-blockchain/discussions/14338
 Is it possible to freeze the thief's coins?

**Reply by shaneo257** [reply]
no, we can't freeze anyone's wallets that we don't control of

**Comment by tig0983** [comment]
Тоже такая же хрень, именно xch1mgda9ll4kst0fh30847tdurjwq9rj6vxumqhxu3xf6r2xt864hdq8s99ut этот адрес вывели 23марта в до этого в январе суммарно 12 манет, НЕУЖЕЛИ ТАКАЯ БОЛЬШАЯ СИСТЕМА НЕ МОЖЕТ ПРИДУМАТЬ ЗАЩИТУ НА ВЫВОД!!!!!!! УЖЕ ВСЕ БИРЖИ ИМЕЮТ 2FA? а CHIA что то блин тупят уже второй год!!!! или ждут когда у них у самих украдут манеты потом начнут чесать затылки что мы дибилы до сих пор GUI не защищен от вывода любым кто захочет

**Reply by Laushd** [reply]
Я думаю это был вирус (мне приходилось отключать защити на 15 минут, и думаю в этот момент вирус сделал своё плохое дело...). У меня не стояла парольная фраза, но и её особого труда не составляет обойти (достаточно на другом компе ввести сид фразу). У тебя стояла дополнительно парольная фраза на кошельке? Как ты думаешь, какие образом злоумышленнику удалось похитить твои монеты?

**Reply by tig0983** [reply]
До первого вывода не стояло, как заметил что манеты отлетают установил, но это ни как не посешало так как GUI то открыт и если есть доступ к пк удаленно то выводи кто хочешь куда хочешь. И было замечено самопроизвольное двидение мышки и так же у меня обычно открыта закладка Фарминга, а на момент когда заметил что что оо не то была открыта закладка кошелька.

**Comment by tig0983** [comment]
Урод вороющий CHIA на адрес (xch1mgda9ll4kst0fh30847tdurjwq9rj6vxumqhxu3xf6r2xt864hdq8s99ut) ты полюбому рано или поздно наткнешься на это сообщение, ЧТОБ у ТЕБЯ ПОЧКИ ОТКАЗАЛИ ТЫ ДОЛГО МУЧИЛСЯ И НА ТВОИ ПОХОРОНЫ ДАЖЕ РОДИТЕЛИ НЕ ПРИШЛИ ОЛЕНЬ ТЫ ЕБАНЫЙ
